packages/mwp-api-proxy-plugin/src/util/send.js

```diff
@@ -271,10 +283,10 @@ export const makeMockRequest = (
 	mockResponseContent,
 	responseMeta
 ) => requestOpts =>
-	Observable.of([
-		makeMockResponse(requestOpts, responseMeta),
-		JSON.stringify(mockResponseContent),
-	]);
+		Observable.of([
```

Comment:
should i run `yarn lint`? if this does it automatically with eslint how do i prevent that?

Reply:
not sure -- sometimes this happens because of a discrepancy between spaces and tabs, so I'd check that first. then, I'd maybe check when the last change was made -- it's possible that the linting config may have changed since then.